discovery-provider/src/api_helpers.py

```diff
@@ -11,4 +18,6 @@ def success_response(response_entity=None, status=200):
     }
 
     response_dictionary['success'] = True
+    response_dictionary['latest_indexed_block'] = int(redis.get(most_recent_indexed_block_redis_key))
+    response_dictionary['latest_chain_block'] = int(redis.get(latest_block_redis_key))
```

Comment:
would be good to make sure there's no case where this breaks eg key doesn't exist or value for key is not int-castable

Reply:
@SidSethi fixed